app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -161,3 +161,37 @@ $color-card-bg: color('white');
     }
   }
 }
+
+// can be added to sections to create slant effect
```

Comment:
This comment should clarify that the affect is applied to the TOP of the element to which you assign this class. We should also include a visual example of this in the style guide dummy app.